code/association_scan/TensorQTL/TensorQTL.ipynb

```diff
@@ -389,6 +389,105 @@
     "    cis_df.to_csv(\"$[_output[1]]\", sep='\\t')"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "[cis_resume]\n",
```

Comment:
@hsun3163 i'll merge it now but I want to make some changes on this one -- i'll elaborate when I reach that point of code review